crates/cargo-wdk/src/actions/new/mod.rs

```diff
@@ -0,0 +1,79 @@
+//! Module that initializes creating a new driver project.
```

Comment:
"intializes creating" is awkward and meaningless. Might wanna reword this sentence.

Reply:
https://github.com/microsoft/windows-drivers-rs/pull/306#discussion_r1983211962